packages/universal-router-sdk/test/fixtures/address.ts

```diff
@@ -0,0 +1,227 @@
+import { ChainId } from '@pancakeswap/chains'
+import { CurrencyAmount, ERC20Token, Pair, Percent, computePairAddress, pancakePairV2ABI } from '@pancakeswap/sdk'
+import { PoolType, SmartRouter, StablePool, V2Pool, V3Pool } from '@pancakeswap/smart-router/evm'
+import {
+  DEPLOYER_ADDRESSES,
+  FeeAmount,
+  Pool,
+  TICK_SPACINGS,
+  TickMath,
+  computePoolAddress,
+  encodeSqrtRatioX96,
+  nearestUsableTick,
+  v3PoolAbi,
+} from '@pancakeswap/v3-sdk'
+import { getPermit2Address } from '@pancakeswap/permit2-sdk'
+import { getUniversalRouterAddress } from '../../src'
+import { Provider, getPublicClient } from './clients'
+import { V2_FACTORY_ADDRESSES } from './constants/addresses'
+import { BUSD, CAKE, ETHER, USDC, USDT, WETH9, WBNB } from './constants/tokens'
+
+const fixtureTokensAddresses = (chainId: ChainId) => {
+  return {
+    ETHER: ETHER.on(chainId),
+    USDC: USDC[chainId],
+    USDT: USDT[chainId],
+    CAKE: CAKE[chainId],
+    WETH: WETH9[chainId],
+    BUSD: BUSD[chainId],
+    WBNB: WBNB[chainId],
+  }
+}
+
+export const getStablePool = async (
+  tokenA: ERC20Token,
+  tokenB: ERC20Token,
+  provider: Provider,
+  liquidity?: bigint,
+): Promise<StablePool> => {
+  const pools = await SmartRouter.getStableCandidatePools({
+    currencyA: tokenA,
+    currencyB: tokenB,
+    onChainProvider: provider,
+  })
+
+  if (!pools.length) throw new ReferenceError(`No Stable Pool found with token ${tokenA.symbol}/${tokenB.symbol}`)
+  const stablePool = pools[0] as StablePool
+
+  if (liquidity) {
+    stablePool.balances[0] = CurrencyAmount.fromRawAmount(stablePool.balances[0].currency, liquidity)
+    stablePool.balances[1] = CurrencyAmount.fromRawAmount(stablePool.balances[1].currency, liquidity)
+  }
+
+  return stablePool
+}
+
+const getPair = (tokenA: ERC20Token, tokenB: ERC20Token, liquidity?: bigint) => {
+  return async (getClient: Provider) => {
+    // eslint-disable-next-line no-console
+    console.assert(tokenA.chainId === tokenB.chainId, 'Invalid token pair')
+    const chainId = tokenA.chainId as ChainId
+    const client = getClient({ chainId })
+    const pairAddress = computePairAddress({ factoryAddress: V2_FACTORY_ADDRESSES[chainId], tokenA, tokenB })
+
+    let reserve0: bigint
+    let reserve1: bigint
+    // @notice: to match off-chain testing ,we can use fixed liquid to match snapshots
+    if (liquidity) {
+      reserve0 = liquidity
+      reserve1 = liquidity
+    } else {
+      ;[reserve0, reserve1] = await client.readContract({
```

Comment:
let's just fix the values in unit test. I think it's safe to remove the fetch from on chain?